corehq/apps/sms/tests/migration.py

```diff
@@ -8,13 +12,16 @@ class BaseMigrationTestCase(TestCase):
 
     def setUp(self):
         self.domain = 'test-sms-sql-migration'
-        self.deleteAllLogs()
+        self.deleteAllObjects()
```

Comment:
yeah i had added that a long time ago for another migration and i don't remember why - it isn't needed. i'll take it out